### ISSUE TITLE
Two finger zoom stuttering [Regression]

### ISSUE THREAD

**Issue by evilpie** [mozilla-mobile/fenix#14148]
## Steps to reproduce
- Goto https://cgit.freedesktop.org/mesa/mesa/log/
- Zoom using two fingers

### Expected behavior
- Smooth zooming without any stutter
- This works in Firefox Beta

### Actual behavior
- Stutter, stepping behavior

### Device information

* Android device: Galaxy S8 (SM-G950F)
* Fenix version: Nightly 200825

I don't know how to bisect Firefox to Android. So far I haven't found the correct builds on https://archive.mozilla.org.

**Comment by evilpie** [comment]
I tried to narrow this down using mozregression and GeckoView and it's kind of hard. I am relatively sure that the first bad build is from 2020-08-18.
The narrowest range that I got is: https://hg.mozilla.org/integration/autoland/pushloghtml?fromchange=069bb8bd2356b4d5738e1cec37bf561c24c0f923&tochange=78205b816ac8fe2730d1fb3fa3b109a5befbfe67 which seems unlikely.

A broader range is: https://hg.mozilla.org/integration/autoland/pushloghtml?fromchange=069bb8bd2356b4d5738e1cec37bf561c24c0f923&tochange=3f301e1e3b7ccc38018abe962cb43a927d7a95a1

This range includes https://bugzilla.mozilla.org/show_bug.cgi?id=1519285 which is quite suspicious to me.
